regtests/ww3_ta1/input_UPD2_U_cap/ww3_uprstr.inp

```diff
@@ -1,57 +1,46 @@
-$           functions(frq,theta) of the update to be applied.
+$ - Starting time in yyyymmdd hhmmss format.                           $
+$                                                                      $
+$ This is the assimilation starting time and has to be the same with   $
+$ the time at the restart.ww3.                                         $
+$                                                                      $
+   19680607 120000                                                     $
+$                                                                      $
+$ -------------------------------------------------------------------- $
+$  UPDN, with N>1 each gridpoint has its own update factor and uses:   $
+$       PRCNTG_CAP - global input for option UPD2+ and is a cap on     $
+$        the maximum SWH correction factor applied to all the          $
+$        gridpoints, as both a multiple or divisor (e.g. cap at 5.0    $
+$        means SWHANL/SWHBKG<=5.0 and SWHANL/SWHBKG>=0.2). The value   $
+$        given should not be less than 1.0                             $
+$       Name of the file with the SWH analysis from the DA system      $
+$        suffix .grbtxt for text out of grib2 file.                    $
+$
+$  UPD2 :: Option 2    The fac(x,y,frq,theta), is calculated at each   $
+$          grid point according to HsBckg and HsAnl                    $
+$          Expected input the Analysis field, grbtxt format            $
+$  UPD3 :: Option 3    The update factor is a surface with the shape   $
+$          of the background spectrum.                                 $
+$          Expected input: the Analysis field, grbtxt format and cap   $
+$          for the last gross check.                                   $
+$  UPD4 :: [NOT INCLUDED in this Version, Just keeping the spot]       $
+$          Option 4    The generalization of the UPD3. The update      $
+$          factor is the sum of surfaces which are applied on the      $
+$          background spectrum.                                        $ 
+$          The algorithm requires the mapping of each partition on the $
+$          individual spectra; the map is used to determine the        $
+$          weighting surfaces.                                         $
+$          Expected input: the Analysis field, grbtxt format and the   $
+$          functions(frq,theta) of the update to be applied.           $
+$ -------------------------------------------------------------------- $
    UPD2
-$
-$ PRCNTG is input for option 1 and it is the percentage of correction
-$applied  to all the gridpoints (e.g. 1.)
-$
-   0.6754
-$
-$ PRCNTG_CAP is global input for option UPD2 and UPD3 and it is a cap on the 
-$ maximun correction applied to all the gridpoints (e.g. 0.5)
-$
-  0.333
-$
-$ Name of the file with the SWH analysis from the DA system            $
-$ suffix .grbtxt for text out of grib2 file.                           $
-$
+   1.333
```

Comment:
The PRCNTG_CAP value has changed, I assume this is to be inline with the new limit "should not be less than 1.0". Why has a limit has been introduced?

Reply:
That's correct. PRCNTG_CAP is used to set an upper limit as 1.0*PRCNTG_CAP and lower limit as 1.0/PRCNTG_CAP in the subroutine CHECK_PRCNTG in ww3_uprstr.ftn

Using this method, setting PRCNTG_CAP less than 1 would give us the opposite behaviour to what is desired (i.e. max < 1, min > 1). This condition was described in a previous revision, but only just spotted that these regtests hadn't been correctly updated

Reply:
Have now added a warning statement if the value of PRCNTG_CAP passed to CHECK_PRCNTG is less than 1

Reply:
I hadn't noticed that it was also used for the lower limit, and it makes sense that it would give the opposite behaviour to what is wanted. All seems good to me.